local_test_settings.py

```diff
@@ -14,6 +14,10 @@
     'molo.core.backends.MoloCASBackend',
 )
 
+INSTALLED_APPS += (
+    'wagtailmedia',
```

Comment:
@maltem-za can you please add `wagtailmedia` in [here](https://github.com/praekelt/molo/blob/develop/molo/core/cookiecutter/scaffold/%7B%7Bcookiecutter.directory%7D%7D/%7B%7Bcookiecutter.app_name%7D%7D/settings/base.py#L60-L72)